README.md

```diff
@@ -139,6 +139,14 @@ Eviction support is limited to the maximum size of the map (defined by `max-size
     
 ***NOTE:*** *QueryCache is always LOCAL to the member and never distributed across Hazelcast Cluster.*
 
+#### Other Properties
+- _cleanup_delay_ - the duration of physical local cache cleanups
+- _cluster_timeout_ - after how many milliseconds the client should stop retrying to establish a cluster connection
```

Comment:
`the number of milliseconds the client should retry to establish a cluster connection`. maybe more clear?

Reply:
```suggestion
- _cluster_timeout_ - the number of milliseconds the client should retry to establish a cluster connection
```

Reply:
works for me